package.json

```diff
@@ -16,9 +16,7 @@
     "format": "prettier --write",
     "lint-ts": "tsc --noEmit --skipLibCheck && eslint --ext .json,.js,.ts,.tsx .",
     "lint-css": "stylelint \"src/**/*.css\"",
-    "link-check": "./scripts/link-check-git-all.sh",
-    "link-check-diff": "./scripts/link-check-git-diff.sh",
-    "link-check-exclude": "./scripts/exclude-links-check.sh"
+    "link-check": "repo-link-check --rootURL='https://www.dvc.org' --fileIncludePattern='**' --linkExcludeFile='scripts/exclude-links.txt' --always-exit-zero"
```

Comment:
a few questions:

1. why do we call it `repo-link-check` not just "link-check"?
2 does it include "link-check-exclude"?
3. --fileIncludePattern='**' - does it literally mean all files? no matter what extension is?

Reply:
I named the package's binary `repo-link-check` because it's the name of the npm package, as `link-check` is already taken by another library.

I meant to come back here to get the exact file patterns to use, `**` was a testing stopgap that does mean all files.

Reply:
Now that I think about it, I can still name the binary 'link-check', as the binary names aren't coupled with package names. I just wanted to avoid confusion for if a user installs `repo-link-check` from NPM and has to run `link-check` as a binary.

Reply:
up to you Roger, I was curious

Reply:
does it include the `link-check-exclude` as well?

Reply:
Do you mean the `exclude-links.txt` file? If so, yes! Excluded links are displayed in the report as always passing with a description of "Excluded". I made sure to re-test that just recently to ensure that behavior still works.
If you mean something else, I must not have been aware of it.

Reply:
I mean a test that check the `exclude-links.txt` file itself, that for each entry we have a match in the repo.

Reply:
Ah, no, I don't currently have that. Since all exclusion patterns are kept in an array, we could track to make sure all are accessed by marking them in the script as the program uses them and then checking if any weren't touched by the end to report them.

Reply:
This feature is in now! Using the CLI with the "--report-unused-patterns" will have the script output any link exclusion patterns that it didn't use. I added this flag to the `link-check-all` invocation, and the results can be seen at the top of the script's output, though when doing so I would recommend piping the output into less or a text file because a full repo report is enough to push the unused patterns out of scroll range on most terminal emulators.